tests/e2e/conftest.py

```diff
@@ -0,0 +1,116 @@
+#
+# Copyright (C) 2024 Red Hat, Inc.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+import json
+import os
+from pathlib import Path
+
+import pytest
+import simple_logger.logger
+from jinja2 import Environment
+from jinja2 import select_autoescape
+from jinja2.loaders import FileSystemLoader
+
+from cli.objects.jira_base import Jira
+
+logger = simple_logger.logger.get_logger(__name__)
+
+
+@pytest.fixture
+def job_dir(build_id):
+    yield Path("/tmp").joinpath(build_id)
+
+
+@pytest.fixture
+def jira_token():
+    res = os.getenv("JIRA_TOKEN")
+    assert res, "JIRA_TOKEN was not found in environment"
+    yield res
+
+
+@pytest.fixture
+def jira_server_url():
+    res = os.getenv("JIRA_SERVER_URL")
+    assert res, "JIRA_SERVER_URL was not found in environment"
+    yield res
+
+
+@pytest.fixture(autouse=True)
+def firewatch_config_path(tmp_path):
+    path = tmp_path.joinpath("firewatch_config.json")
+    path.parent.mkdir(exist_ok=True, parents=True)
+    path.write_text(
+        json.dumps(
+            {
+                "failure_rules": [
+                    {
+                        "step": "ipi-conf*",
+                        "failure_type": "all",
+                        "classification": "Infrastructure Provisioning - Cluster",
+                        "group": {"name": "cluster", "priority": 1},
+                        "jira_additional_labels": ["!default"],
+                    },
+                ],
+            },
+        ),
+    )
+    logger.info(f"wrote firewatch config to {path.as_uri()}")
```

Comment:
you have `logger.success` which fit here if you want to use it.

Reply:
`logger.success` is not used anywhere else. should this be refactored separately?

Reply:
up to you if you want to use it or not, I'm just notifying you that it is existing and fits here.